src/Tag/ManagerInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace FOS\HttpCache\Tag;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Implementing classes are responsible for associating tags with cache entries
+ * and invalidating tags (and removing cache entries).
+ */
+interface ManagerInterface extends InvalidatorInterface
```

Comment:
Not sure that is a better name.. this is also an invalidator and there already exists a `StorageInterface`.